client/src/main/scala/org/apache/celeborn/client/LifecycleManager.scala

```diff
@@ -1655,6 +1693,25 @@ class LifecycleManager(val appUniqueId: String, val conf: CelebornConf) extends
     }
   }
 
+  private def batchRequestMasterUnregisterShuffles(message: PbBatchUnregisterShuffles)
+      : PbBatchUnregisterShuffleResponses = {
+    try {
+      logInfo(s"AskSync UnregisterShuffle for ${message.getShuffleIdsList}")
+      masterClient.askSync[PbBatchUnregisterShuffleResponses](
+        message,
+        classOf[PbBatchUnregisterShuffleResponses])
+    } catch {
+      case e: Exception =>
+        logError(s"AskSync UnregisterShuffle for ${message.getShuffleIdsList} failed.", e)
+        val map = JavaUtils.newConcurrentHashMap[Integer, StatusCode]()
+        val shuffleIds = message.getShuffleIdsList
```

Comment:
Only for subset where `PbUnregisterShuffleResponsesInfo.status` is failed ?

Reply:
In this case, all subsets are anomalous and there is no need for repeated checks.The part related to filling the map has been removed.